cmd/fts/cmd/run.go

```diff
@@ -0,0 +1,170 @@
+package cmd
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/gosuri/uiprogress"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	dryRun         bool
+	mainnet        bool
+	maxStagedBytes int64
+	maxDealBytes   int64
+	minDealBytes   int64
+	hiddenFiles    bool
+	taskFolder     string
+	resultsOut     string
+	concurrent     int64
+	ipfsrevproxy   string
+	retryErrors    bool
+)
+
+func init() {
+	runCmd.Flags().BoolVarP(&dryRun, "dry-run", "d", false, "Run through steps without pushing data to Powergate API.")
+	runCmd.Flags().BoolVarP(&mainnet, "mainnet", "n", false, "Sets staging limits based on localnet or mainnet.")
+	runCmd.Flags().BoolVarP(&retryErrors, "retry", "e", false, "Retry tasks with error status")
+	runCmd.Flags().StringVar(&taskFolder, "folder", "", "Folder with organized tasks of directories or files")
+	runCmd.Flags().BoolVarP(&hiddenFiles, "all", "a", false, "Include hidden files & folders from top level folder")
+	runCmd.Flags().StringVar(&ipfsrevproxy, "ipfsrevproxy", "127.0.0.1:6002", "Powergate IPFS reverse proxy multiaddr")
+	runCmd.Flags().StringVarP(&resultsOut, "results", "r", "results.json", "The location to store intermediate and final results.")
+
+	// Not included in the public commands
```

Comment:
below are mostly the dev user settings, not really helpful to have in the public interface